humans/migrations/0010_auto_20170519_1437.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration is not valid since `import languages.fields` is not a dependency of the project. An option would be to delete these migrations, and generate a new one with the real changes to the schema.